src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -198,6 +194,7 @@ void EIRPlantLoopHeatPump::resetReportingVariables()
     this->fractionalDefrostTime = 0.0;
     this->partLoadRatio = 0.0;
     this->cyclingRatio = 0.0;
+    this->defrostPowerMultiplier = 1.0;
```

Comment:
Since defrost calculation is refactored to a new function I added a new member variable `defrostPowerMultiplier` for use else where.